settings.gradle.kts

```diff
@@ -20,6 +20,7 @@ dependencyResolutionManagement {
 rootProject.name = "sentry-kotlin-multiplatform-sdk"
 
 include(":sentry-kotlin-multiplatform")
+includeBuild("plugin-build")
```

Comment:
nit: I think you'd need to define `dependencySubstitution` block so you don't have to publish to local maven every time, and just develop against the included build, like here: https://github.com/getsentry/sentry-android-gradle-plugin/blob/f9b82ddce3af1a21eec54da8b1c02e36c063939a/settings.gradle.kts#L37-L41

Reply:
it already works even without `dependencySubstitution`. 

I don't need to do a local publish during dev

Reply:
oh, ok, I didn't know that works! probably when using the new `plugins { }` block, it just works